test/recipes/99-test_fuzz_cmp.t

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env perl
+# Copyright 2016-2020 The OpenSSL Project Authors. All Rights Reserved.
+#
+# Licensed under the Apache License 2.0 (the "License").  You may not use
+# this file except in compliance with the License.  You can obtain a copy
+# in the file LICENSE in the source distribution or at
+# https://www.openssl.org/source/license.html
+
+use strict;
+use warnings;
+
+use OpenSSL::Test qw/:DEFAULT srctop_file/;
+use OpenSSL::Test::Utils;
+
+setup("test_fuzz_cmp");
+
+if (disabled("cmp")) {
+    plan skip_all => "This test requires cmp support";
```

Comment:
`plan skip_all` exits the recipe immediately, so you can safely have something like this:

``` perl
setup('test_fuzz_cmp');

plan skip_all => "This test requires cmp support" if disabled('cmp');

plan tests => ...
```